refinery/ui/source/js/tool-launch/ctrls/tool-launch-button-ctrl.js

```diff
@@ -35,13 +36,12 @@
     toolSelectService,
     $uibModal,
     visualizationService,
-    $rootScope,
     $window
   ) {
     var vm = this;
     vm.launchTool = launchTool;
     vm.needMoreNodes = needMoreNodes;
-    vm.userIsAnonymous = settings.djangoApp.userId !== undefined;
+    vm.userIsAnonymous = settings.djangoApp.userId === undefined;
```

Comment:
Would it make more sense to follow the existing usage of `isLoggedIn` (https://github.com/refinery-platform/refinery-platform/search?q=isLoggedIn&unscoped_q=isLoggedIn) and perhaps factor this out into a re-usable function or service?